py/build.gradle

```diff
@@ -111,8 +107,8 @@ task distPython(type: Exec, dependsOn: checkPythonEnv) {
         }
 
         copy {
-            from "${configurations.compile.join(',')}"
```

Comment:
PR 195 has higher priority. Build time is not bottle neck right now. Keep this change opened.